### ISSUE TITLE
Questions....

### ISSUE THREAD

**Issue by UnderAverageGuy** [mhkamal15/DarkAges#1]
IK its just in the beggining or a WIP but if you are writing to a .py as your backdoor you will need something to run it on every device... although making a .exe would be complicated It's pointless to have a .py backup file unless that user has no common sense and puts their password in to allow a sussy file to download required modules and run on the terminal.

**Comment by mhkamal15** [comment]
Yeah I've been browsing on how to make it a .exe file, but I have yet to make much progress.

**Comment by TopSlime999** [comment]
even once it is a .exe wont it have to have admin or something to execute? either that or some method of getting into the startup folder....

**Comment by mhkamal15** [comment]
W... I... P...
I also find it ironic that your virus has the same issues you point out.

**Comment by UnderAverageGuy** [comment]
ik, these are problems I have thought about but not the deep into just tryna help find improvment and issues for the furture man...

**Comment by UnderAverageGuy** [comment]
I wonder if it would be possile to have a second part of the program that runs a command or gains access to sudo that could possibiliy hide the terminal or freeze the screen then randomize the new exe after storing it somewhere secure or in the startup folder? no idea if it would work but its interesting.

*using sudo to either import pyinstaller or already have it contained within the exe? then using it on the file automatically

**Comment by mhkamal15** [comment]
> ik, these are problems I have thought about but not the deep into just tryna help find improvment and issues for the furture man...

I was jk man. Yeah IK it's completely unrealistic this thing could run. I've been playing with py2exe lately, and all I've been getting is permission-related errors, so I think there probably does need to be a function to escalate admin privileges.

Funny because I thought the same exact thing about using that tactic. I think it might be possible to use described functions to hide the terminal, freeze the screen, randomize the new .exe, and probably just run a command or gain access to sudo. Potential collab. Would take some to research though. I'll get to it sometime.

**Comment by UnderAverageGuy** [comment]
PRIVILEGE escalation is probably pretty difficult, faking a terminal could be an option, other than that we could have the program wait until the user opens a sudo terminal but there is no 100% chance they do that. it would be pretty hard to steal sudo privileges otherwise...

**Comment by mhkamal15** [comment]
It probably is... no attempt at trying is too bad though.
